[ADD] prln_aging_payable_report

mikevhe18 opened the pull request.

Commit:
Creating a first structure module directory

Commit:
Creating a first structure module directory

andhit-r commented:
@mikevhe18 Per report nanti dibuat satu module. Jadi nama modulnya disesuaikan. nama modulnya dpnnya prln_. Ga usah ada opnsyn nya mike. Kepanjangan

andhit-r commented:
@mikevhe18 satu modul satu PR yak mike. Jangan digabung

mikevhe18 commented:
Ohh satu report satu module yak mas..
Oke deh nanti diganti kalau gitu

Commit:
1. Change a module name to prln_aging_payable_report
2. Create a wizard for print report
3. Create a template aging_payable_account report(Just a template, coding not yet include)

Commit:
1. Change a module name to prln_aging_payable_report
2. Create a wizard for print a report
3. Create a template report(Just a template, Coding is not yet include)

Commit:
Merge branch '6.1' of https://github.com/mikevhe18/prln into 6.1

Commit:
1. Delete a previous module(aeroo_opnsyn_pralon)

Commit:
Fixing identation using retab

andhit-r commented:
@mikevhe18 gw dah buat supaya maintener-quality-toolsnya pakai punya open-synergy. Rebase dulu coding lo dengan kode terakhir.

andhit-r commented:
@mikevhe18 jgn lupa rebase. Gw banyak perubahan

Commit:
1. Templating report aging account payable

Commit:
Merge remote-tracking branch 'opnsyn_pralon/6.1' into 6.1

Commit:
1. Fixing identation

Commit:
* Fixing error  travis
